source/_components/somfy_mylink.markdown

```diff
@@ -0,0 +1,67 @@
+---
+layout: page
+title: "Somfy MyLink"
+description: "Instructions on how to integrate Somfy MyLink devices with Home Assistant."
+date: 2019-03-29 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tahoma.png
+ha_category:
+  - Hub
+  - Cover
+ha_release: 0.92
+ha_iot_class: "Assumed State"
+---
+
+The `Somfy MyLink` component platform is used as an interface to a compatible Somfy MyLink hub utilizing the `Synergy` API. It adds covers and scenes from the Somfy MyLink platform.
```

Comment:
Does it add scenes as cover entities?

Reply:
The `Somfy MyLink` integration is used...

Reply:
@MartinHjelmare - it doesnt actually add scenes as I stripped out that platform from the PR. I'll remove that from the docs.
What do you mean by 'The Somfy MyLink integration is used...'?

Reply:
We prefer talking about the integration as a whole instead of component and platforms. That's our new direction.

Reply:
Ohhh right sorry, I understand now 👍 I thought you were saying that name was taken by another integration!